spec/defines/vhost_spec.rb

```diff
@@ -383,6 +383,7 @@
           'passenger_app_env'           => 'test',
           'passenger_ruby'              => '/usr/bin/ruby1.9.1',
           'passenger_min_instances'     => '1',
+          'passenger_max_requests'      => '1000',
```

Comment:
It's similar to this configuration from apache.
https://httpd.apache.org/docs/2.4/en/mod/mpm_common.html#maxconnectionsperchild
Therefore you should set a non-zero value.